### DISCUSSION TITLE
Windows docker image .net core throws error

### DISCUSSION THREAD

**Discussion by atulthummar** [dotnet/dotnet-docker#3130] [General]
**C:\WINDOWS\system32>kubectl logs jobprocessor-function-cfb955886-fmqbd -p**
It was not possible to find any installed .NET Core SDKs
Did you mean to run .NET Core SDK commands? Install a .NET Core SDK from:
https://aka.ms/dotnet-download

getting above while checking pod log and pod status CrashLoopBackOff

Even though i have installed - FROM mcr.microsoft.com/dotnet/core/runtime:3.1-nanoserver-1803 AS base

**Comment by atulthummar** [comment]
C:\WINDOWS\system32>kubectl logs jobprocessor-function-5c57b6cb95-4hk5n
Hosting environment: Production
Content root path: C:\app
Now listening on: http://[::]:80
Application started. Press Ctrl+C to shut down.

**Comment by MichaelSimons** [comment]
Can you share what you Dockerfile looks like?  Usually this error is caused by one of two things.  You are either trying to use the .NET CLI in a runtime image which does not contain the CLI.  Use the dotnet/sdk images if this is the case.  The other cause is that you are trying to run an application via `dotnet <YourAppName>` but either the app doesn't exist as expected or you are using a linux image (a case sensitive environment) and the casing is incorrect.

**Reply by atulthummar** [reply]
#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.

FROM mcr.microsoft.com/azure-functions/dotnet:3.0-nanoserver-1809 AS base
WORKDIR /app
EXPOSE 80

FROM mcr.microsoft.com/dotnet/sdk:3.1-nanoserver-1809 AS build
WORKDIR /src
COPY ["FunctionApp.csproj", "FunctionApp/"]
RUN dotnet restore "FunctionApp/FunctionApp.csproj"
COPY . .
WORKDIR "/src/FunctionApp"
RUN dotnet build "FunctionApp.csproj" -c Release -o /app/build

FROM build AS publish
RUN dotnet publish "FunctionApp.csproj" -c Release -o /app/publish

FROM base AS final
WORKDIR /app
COPY --from=publish /app/publish .

**Reply by MichaelSimons** [reply]
Do you have the command that is being used to run the container?

**Reply by atulthummar** [reply]
I have used Azure function 3.1 its on service bus trigger - I have to run on window docker image
If i pass - ENTRYPOINT ["dotnet", "FunctionApp.dll"]
Then pod logs shows message  as "It was not possible to find any installed .NET Core SDKs
Did you mean to run .NET Core SDK commands? Install a .NET Core SDK from:
https://aka.ms/dotnet-download" 

If not then it shows 
Content root path: C:\app
Now listening on: http://[::]:80
Application started. Press Ctrl+C to shut down.

.Net core function runtime should be required as its already install with base image FROM mcr.microsoft.com/azure-functions/dotnet:3.0-nanoserver-1809 AS base

**Reply by atulthummar** [reply]
Any input ?
